nservicebus/transactional-session/index.md

```diff
@@ -84,7 +84,7 @@ The default commit delay increment is set to `Timespan.FromSeconds(2)`and cannot
 
 #### Metadata
 
-It is possible to add custom headers to the control message that is used to settle the transaction outcome. Custom headers can be set by adding them to the metadata when opening the session. These headers can be accessed by a [custom behavior](/nservicebus/pipeline/manipulate-with-behaviors.md#add-a-new-step) in the `TransportReceive` part of the pipeline.
+It is possible to add metadata (e.g. tenant information) via custom headers to the transactional session control message sent. These headers can be accessed by a [custom behavior](/nservicebus/pipeline/manipulate-with-behaviors.md#add-a-new-step) when the control message is received in the `TransportReceive` part of the pipeline.
```

Comment:
I think this is good enough. I was thinking about adding something like

"This is mostly used for extensibility of the persistence-specific implementations" but figured this is not really a hint that is required in this context. The only other alternative for me would be to remove this documentation section entirely, which, I feel, is also not a good idea.

Reply:
Agreed. The task force has already decided not to emphasize this API. The tenant information is also the only use case I can think of right now, so I agree that this is good enough now.